config/helm/chart/default/templates/_helpers.tpl

```diff
@@ -114,7 +114,7 @@ Check if default image is used
 {{- if .Values.image -}}
 	{{- printf "%s" .Values.image -}}
 {{- else -}}
-	{{- if eq .Values.platform "google" -}}
+	{{- if eq .Values.platform "google-marketplace" -}}
```

Comment:
also, the google-autopilot changes in this PR would break things, so modify the `schema.yaml` in a way that it adds a `bool` value that is used to check if its necessary to apply the `application.yaml` 

and then the `google-marketplace` as a platform makes no sense, it should be `google-standard` to match with `google-autopilot`

Reply:
It's currently not possible to deploy to a GKE Autopilot cluster from a Marketplace listing.

Let's stick with `google-marketplace` and `gke-autopilot` as platforms name. `google-marketplace` should not be mentioned in the `values.yaml`, only platforms that should be set by the user needs to be documented there.

Reply:
done